app/assets/stylesheets/blacklight/_pagination.scss

```diff
@@ -9,3 +9,12 @@
   text-align: center ;
 }
 
+
+.pagination-xs {
+  li {
+    a, span {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line